src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './Introduce.scss';
+import { Link, useNavigate } from 'react-router-dom';
+
+function Introduce() {
+  const clamp = '>wecode';
+
+  const Navigate = useNavigate();
```

Comment:
```suggestion
  const navigate = useNavigate();
```

첫 글자가 대문자로 된건 컴포넌트에만 적용이 된다고 생각해주세요!